Tests/Rules/PlaceCloseBrace.tests.ps1

```diff
@@ -80,19 +80,31 @@ $hashtable = @{a = 1; b = 2}
 
     Context "When there is a multi-line hashtable" {
         BeforeAll {
-            $def = @'
-$hashtable = @{
-    a = 1
-    b = 2}
-'@
+
             $ruleConfiguration.'NoEmptyLineBefore' = $false
             $ruleConfiguration.'IgnoreOneLineBlock' = $true
             $ruleConfiguration.'NewLineAfter' = $false
-            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
         }
 
         It "Should find a violation" {
+            $def = @'
+            $hashtable = @{
+                a = 1
+                b = 2}
+'@
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
+            $violations.Count | Should -Be 1
+        }
+
+        It "Should not crash when hashtable is defined on first token" {
+            $def = @'
```

Comment:
I think this can also be written as follows:
```
$def = "@{ `n Key = 1 }"
```
and it will cause the same error

Reply:
and it's a very specific set of characters. This `"@{key=1}"` doesn't cause the problem but `"@{``nkey=1}"` does

Reply:
Ok, I can change it to that

Reply:
Done now